### PR TITLE
Part2

### PR DESCRIPTION
**Pull request by shaefelinger**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -108,9 +108,48 @@ struct CarWash
...
 
+struct Person
+{
+    int age;
+	int height;
+	float hairLength;
+	float GPA;
+	unsigned int SATScore;
+    int distanceTraveled;
+
```
Comment (matkatmusic) [inline main.cpp]: Fix this indentation
- [x] revise

File: main.cpp [hunk 2]
```diff
@@ -108,9 +108,48 @@ struct CarWash
...
+    {
+        //
+    }
+};
 
+void Person::run(bool startWithLeftFoot)
+{
+    Foot leftFoot;
+    Foot rightFoot;
+    if(startWithLeftFoot == true)
```
Comment (matkatmusic) [inline main.cpp]: These were not declared in the code in the video.  That means they are member variables of the `Person` class.
Move these declarations into the `Person` class
- [x] revise

File: main.cpp [hunk 3]
```diff
@@ -216,293 +246,292 @@ struct Computer
...
-    // activate smart tv
     void activateSmartTv(int menuItem = 0);
 };
 
+void Tv::changeChannel(int channelNumber, bool useRemoteControl)
+{
+    if (useRemoteControl)
+    {
+        channelNumber++;
+    }
```
Comment (matkatmusic) [inline main.cpp]: Everywhere that you have `i++`, `i--`, `i -= 1;` or `i += 1;`, you should use `pre-increment/decrement`. 
You are currently doing `post-increment/decrement`.  
you can think of post- and pre- increment like this:
```
int preIncrement(int& i)
{
    i += 1;
    return i; //both the returned value AND i have changed
}
​
int postIncrement(int& i)
{
    int oldVal = i;
    i += 1;
    return oldVal; //i has changed but the value being returned does not reflect this change.
}
```
more information can be found here explaining why `pre-increment/decrement` is preferred over `post-increment/decrement`: https://en.cppreference.com/w/cpp/language/operator_incdec

File: main.cpp [hunk 4]
```diff
@@ -216,293 +246,292 @@ struct Computer
...
+void Tv::changeVolume(float newVolume, bool useRemoteControl)
+{
+    newVolume = 3.3f;
+    useRemoteControl = false;
+}
+
+void Tv::activateSmartTv(int menuItem)
+{
+    menuItem++;
+}
```
Comment (matkatmusic) [inline main.cpp]: use `pre-increment`
- [x] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 1 above)
Comment (shaefelinger) [reply main.cpp:119]: ok, this is the way it looks in my replit. any idea what might cause this? i copy/pasted it from my notes, maybe there is the problem??
image

File: main.cpp (hunk 3 above)
Comment (shaefelinger) [reply main.cpp]: good to know...

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**